test/NuGet.Tests.Apex/NuGet.Tests.Apex/Utility/CommonUtility.cs

```diff
@@ -149,6 +154,17 @@ public static SimpleTestPackageContext CreatePackage(string packageName, string
             return package;
         }
 
+        public static SimpleTestPackageContext CreateCustomPackage(string packageName, string packageVersion, string requestAdditionalContent)
```

Comment:
nit: CustomPackage is not a super descriptive name.  A better name would call out what it's compatible with.

Reply:
Refactored it and rename to `CreateNetCorePackage`.